code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -396,21 +396,18 @@
 	cooldown_per_use = 10
 
 /obj/item/integrated_circuit/input/turfscan/do_work()
-	var/atom/movable/H = get_pin_data_as_type(IC_INPUT, 1, /atom)
+	var/turf/H = get_pin_data_as_type(IC_INPUT, 1, /turf)
 	var/turf/T = get_turf(src)
-	var/turf/E = get_turf(H)
 	if(!istype(H)) //Invalid input
 		return
 
 	if(H in view(T)) // This is a camera. It can't examine thngs,that it can't see.
 		var/list/cont = new()
-		if(E.contents.len)
-			for(var/i = 1 to E.contents.len)
-				var/atom/U = E.contents[i]
-				cont += WEAKREF(U)
+		for(var/obj/U in H)
+			cont += WEAKREF(U)
```

Comment:
You limited it to objects when it wasn't limited before?

Reply:
Yes, did you see what I said? Things like /atom/movable/lighting_object show up on the list.
I can't imagine that that would be intended. Can you think of something that isn't an /obj that you would want to use this for? And if you can, how should we disable things we don't seem usable (Like the lighting atom)?

Reply:
mobs are movable atoms. If you want to add those do a seperate loop with `for(var/mob/M in H)`

Reply:
Will do.